axon-application-development/query-handling/exception-handling-1.md

```diff
@@ -0,0 +1,12 @@
+# Exception Handling
```

Comment:
I've seen several files being duplicated up till now, same for this guy. Once as `exception-handling-1.md` and another occurrence called `exception-handling.md`, both in the `query-handling` directory. Is this intentional?